tests/utils/test_tickets.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+from invenio_testing import InvenioTestCase
+
+import mock
+
+
+# FIXME(jacquerie): should be in an helpers file.
+rt_cfg = {
+    'CFG_BIBCATALOG_SYSTEM_RT_URL': 'url',
+    'CFG_BIBCATALOG_SYSTEM_RT_DEFAULT_USER': 'default_login',
+    'CFG_BIBCATALOG_SYSTEM_RT_DEFAULT_PWD': 'default_password'
+}
+
+
+class TicketsTests(InvenioTestCase):
+
+    """Tests for the RT ticketing system."""
+
+    @mock.patch('inspirehep.utils.tickets.cfg', rt_cfg)
+    @mock.patch('inspirehep.utils.tickets.rt.Rt')
+    def test_get_instance(self, Rt):
```

Comment:
A bad test, because it's a change-detector test: http://googletesting.blogspot.ch/2015/01/testing-on-toilet-change-detector-tests.html.